ui/components/app/deposit-popover/index.scss

```
@@ -0,0 +1,14 @@
+.popover--deposit {
+  width: 100% !important;
+  min-width: 328px;
+  max-width: 850px;
+}
```

This change suggestion is to fix the missing padding on the small screen version (when you click on the extension without expanding the view, the popover padding is missing because of the `width: 100% !important`
```suggestion
.popover--deposit {
  min-width: 328px;
  max-width: 850px;
  @include screen-md-min {
    width: 100% !important;
  }
}
```